error_reporting/setup.py

```diff
@@ -51,14 +51,14 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.24.1, < 0.25dev',
+    'google-cloud-core >= 0.25.0, < 0.26dev',
     'google-cloud-logging >= 1.0.0, < 2.0dev',
```

Comment:
Surely we didn't introduce breaking changes in `logging` without bumping its major version. :)